bot/exts/info/information.py

```diff
@@ -36,47 +29,60 @@ def __init__(self, bot: Bot):
-        - Do staff roles have explicit read allow permissions?
+    @staticmethod
+    def get_channel_type_counts(guild: Guild) -> DefaultDict[str, int]:
+        """Return the total amounts of the various types of channels in `guild`."""
+        channel_counter = defaultdict(int)
 
-        If the answer to both of these questions is yes, it's a staff channel.
-        """
-        channel_ids = set()
         for channel in guild.channels:
-            if channel.type is ChannelType.category:
-                continue
-
-            everyone_can_read = self.role_can_read(channel, guild.default_role)
-
-            for role in constants.STAFF_ROLES:
-                role_can_read = self.role_can_read(channel, guild.get_role(role))
-                if role_can_read and not everyone_can_read:
-                    channel_ids.add(channel.id)
-                    break
+            if Information.is_staff_channel(guild, channel):
+                channel_counter["staff"] += 1
+            else:
+                channel_counter[str(channel.type)] += 1
 
-        return len(channel_ids)
+        return channel_counter
 
     @staticmethod
-    def get_channel_type_counts(guild: Guild) -> str:
-        """Return the total amounts of the various types of channels in `guild`."""
-        channel_counter = Counter(c.type for c in guild.channels)
-        channel_type_list = []
-        for channel, count in channel_counter.items():
-            channel_type = str(channel).title()
-            channel_type_list.append(f"{channel_type} channels: {count}")
-
-        channel_type_list = sorted(channel_type_list)
-        return "\n".join(channel_type_list)
+    def get_member_counts(guild: Guild) -> Dict[str, int]:
+        """Return the total number of members for certain roles in `guild`."""
+        roles = (
+            guild.get_role(role_id) for role_id in (
+                constants.Roles.helpers, constants.Roles.moderators, constants.Roles.admins,
+                constants.Roles.owners, constants.Roles.contributors,
+            )
+        )
+        return {role.name.title(): len(role.members) for role in roles}
+
+    def get_extended_server_info(self, guild: Guild) -> str:
+        """Return additional server info only visible in moderation channels."""
+        talentpool_count = len(self.bot.get_cog("Talentpool").watched_users)
+        bb_count = len(self.bot.get_cog("Big Brother").watched_users)
+
+        defcon_cog = self.bot.get_cog("Defcon")
+        defcon_status = "Enabled" if defcon_cog.enabled else "Disabled"
+        defcon_days = defcon_cog.days.days if defcon_cog.enabled else "-"
+        python_general = self.bot.get_channel(constants.Channels.python_discussion)
+
+        return textwrap.dedent(f"""
+            Nominated: {talentpool_count}
+            BB-watched: {bb_count}
+            Defcon status: {defcon_status}
+            Defcon days: {defcon_days}
```

Comment:
Considering the defcon cog won't stay a part of this bot for much longer, I'm not sure including this is a good idea. But maybe it can stay for now.

Reply:
I'll keep it in for now.